contrib/networkmanager-openconnect/files/sysusers.conf

```diff
@@ -0,0 +1,3 @@
+# Create nm-openconnect system user
+
+u nm-openconnect - "nm-openconnect user" /var/lib/NetworkManager /usr/bin/nologin
```

Comment:
is there a way to pick the user for this? we generally aim for having underscore-prefixed system users/groups if we can help it

Reply:
looks like we'd need to patch `NM_OPENCONNECT_USER` in the code, and then the polkit rulesets